### ISSUE TITLE
suggestion on pagination

### ISSUE THREAD

**Issue by cantoute** [kepelrs/nestjs-prisma-crud#272]
It's a common implementation to use pager logic but I'm not super fond of it.
I don't mind handling `skip=pageSize*page` front end side.  One simple use case is when user changes pageSize, trying to guess what page we should go to, and where in the page it'll find the records that where visible at time of page size change. (so easy to simply reuse the current `skip`)
Something I really appreciate to have in the api response is a `hasMore` or `endOfRecords` boolean 

As I'm considering using this crud module on a personal project, I thought I could offer to contribute by first adding a few things on the roadmap.

- [] Add a `hasMore` boolean in the response
- [] Add the ability to use skip/take pagination logic exposing take skip in all responses, if `page` and `take/skip` present in request )
- [] Take advantage of cursor logic (

**Comment by kepelrs** [comment]
I agree that exposing take/skip enables more flexibility in certain scenarios. Will keep it in mind during work planned for upcoming months.
